tests/opwen_email_server/utils/test_unique.py

```diff
@@ -15,6 +16,7 @@ def test_unique(self):
 
 
 class NewGuidTests(TestCase):
+
```

Comment:
Spurious formatting changes make this harder to review than necessary. Could you revert all the non-essential changes? Even helps to keep the merged diff more clean/minimal. TIA!

Reply:
I reverted code formatting.